spark/src/test/java/org/opensearch/sql/spark/asyncquery/IndexQuerySpecTest.java

```diff
@@ -469,50 +427,35 @@ public EMRServerlessClient getClient() {
                       new CreateAsyncQueryRequest(mockDS.query, DATASOURCE, LangType.SQL, null));
 
               // 2. fetch result
-              assertEquals(
-                  "SUCCESS",
-                  asyncQueryExecutorService
-                      .getAsyncQueryResults(response.getQueryId())
-                      .getStatus());
-
+              AsyncQueryExecutionResponse asyncQueryExecutionResponse =
+                  asyncQueryExecutorService.getAsyncQueryResults(response.getQueryId());
+              assertEquals("SUCCESS", asyncQueryExecutionResponse.getStatus());
```

Comment:
nit, what is the difference? any consideration?

Reply:
Aaah...changed as part of refactoring. I was storing response in a separate variable for other test cases to assert Error.